foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -112,14 +110,14 @@ if (app != null) {
 
 ## Track app actions
 
-Actions are events that occur in your app. You can use this API to track and measure an action. Each action has one or more corresponding metrics that are incremented each time the event occurs. For example, you might call this API for each new subscription each time an article is viewed, or each time a level is completed.
+Actions are events that occur in your application. You can use the `trackAction` method to track and measure an action. Each action has one or more corresponding metrics that are incremented each time the event occurs. For example, you can use an action to track new subscriptions, every time an article is viewed, or every time a level is completed.
 
 {% hint style="warning" %}
-Call this API when an event that you want to track occurs. In addition to the action name, you can send additional context data with each track action call.
+You want to use the `trackAction` method when you want to track an occurring event. In addition to the action name, you can send additional context data with each track action call.
```

Comment:
this hint sounds wonky.  the second sentence sounds redundant with multiple "addition"s